packages/freeimage/swab.patch

```diff
@@ -0,0 +1,47 @@
+--- <null>
++++ b/Source/LibRawLite/internal/swab.cpp
+@@ -0,0 +1,39 @@
++/*
++ * Copyright (C) 2017 The Android Open Source Project
```

Comment:
Nicely done with the swab patch!

Since swab.cpp is licensed under BSD 2-Clause we also need to specify this in `TERMUX_PKG_LICENSE`, and add this copyright header as a `license-bsd-2-clause.txt` file to the package (as done for example here [[1]](https://github.com/termux/termux-packages/blob/master/packages/libandroid-spawn/LICENSE), [[2]](https://github.com/termux/termux-packages/blob/09d4c3a6c9a3f3c47408d823b64def789481daa9/packages/libandroid-spawn/build.sh#L13)). 
This is a bit annoying but needed if we want to comply with the BSD license.

Reply:
I updated it.